prime-router/docs/releases/next-release.md

```diff
@@ -0,0 +1,14 @@
+#  Hub Release Notes
+
+*NO DATE SET*
+
+## General useful links:
+
+- All Schemas are documented here:  [Link to detailed schema dictionaries](../schema_documentation)
+- The Hub API is documented here: [Hub OpenApi Spec](../openapi.yml)
+- [Click here for all release notes](../releases)
+
+## Changes to the Schemas, including for SimpleReport:
+
+Added optional field `Corrected_result_ID`, a pointer/link to the unique Result_ID of a previously submitted result.  Usually blank. Or, if an item modifies/corrects a prior item, this field holds the ResultID of the prior item.
```

Comment:
If I’m starting to understand how everything works correctly, the field is only named `Corrected_result_ID` for input from SimpleReport (i.e. the `primedatainput/pdi-covid-19` schema) and all other schemas (so, basically all receivers/services) will see this as `previous_message_id`, right?

If so, it might be worth clarifying that here; this doesn’t mention `previous_message_id`, but it’s the one that *most* people/tools interacting with DataHub will actually see.

Reply:
@Mr0grog , Not 100% sure, but I think that's not quite right  - the 'previous_message_id' name is internal - its the [ineffable name that no human research can discover](https://poets.org/poem/naming-cats)

At least for CSVs, I believe each incoming and outgoing schema must specify the externally visible`name` in order to work.

Reply:
> each incoming and outgoing schema must specify the externally visiblename in order to work.

Ahhhhhhh, ok. So the effective change here is that we are allowing `Corrected_result_ID` in SimpleReport input, but not necessarily *sending* that field anywhere yet?